### ISSUE TITLE
GPU Fans shutdown on Save Configuration

### ISSUE THREAD

**Issue by jugit** [Rem0o/FanControl.Releases#111]
Cool program, but it scared me for a sec. I hid the GPU cards since i control the fans with Afterburner and wanted mainly to control the chassis fans. Using an example sensor file to set the temp over the minimum temp, the fans were running at almost 100% RPM as expected. But when i saved this configuration, I noticed all my GPU fans went to 0 RPM. I had to trigger them back on quickly in Afterburner. Tried this a second time and happened again.

**Comment by Rem0o** [comment]
Could you clarify the state of the application and afterburner when that happens? What kind of GPU?

Just tried disabling the GPU in FanControl and activating it in AfterBurner at 50%.  Saved the profile in FanControl, it didn't turn off the gpu fan or anything like that.

**Comment by jugit** [comment]
Ok, so I think I know what's going on here. The app lists 3 fans for the 2 GPUs I have on the system. They're all at default config of 0%. I only configured a curve for the chassis fans. So when I recently rebooted, I noticed Afterburner would bring up my preset config and throw my fans to 95%. Then Fan Control opened and killed it setting them to 0 RPM. So it seems that if they aren't configured in the app, they are set to the default values of zero percent. I imagine that when I previously saved the configuration in the app, it applied the saved config, resetting the GPU fans to zero. 
Is there a way to exclude the GPU fans from the app's control or any fan that has not been initially configured? Hiding doesn't seem to do the trick.
Not sure if the screen capture helps, since I already set a flat curve for the GPUs, but where it shows 92%, they were first detected as 0% and I think that was the issue.

![2021-01-09 19_25_40-Fan Control V62 (config2 json)](https://user-images.githubusercontent.com/43105673/104111509-91df8a00-52b0-11eb-8a81-9f3e77236288.png)
